web-interface/api/logic.py

```diff
@@ -101,13 +101,33 @@ def AppendDiskInfo(diskMountPoint):
                  })
     pass
 
-# Going to be updating to increase efficiency of nmap scans.
+
+# Create a list of ports and services, and create a class to define ports and services.
+ServiceList = []
+class PortInfo:
+    def __init__(self, port, service, IsActive):
```

Comment:
For cleaner code, the properties of the class and thus the parameters of the constructor, should reflect the json-output

{
    "PortNumber": portNumber,
    "ServiceName": serviceName,
    "IsActive": isActive
}

As you can see, port & service are not according to this 'rule'